core/federated/RTI/rti.Dockerfile

```diff
@@ -1,13 +1,32 @@
-# Docker file for building the image of the rti
-FROM alpine:latest
+# base image from which all other stages derive
+ARG BASEIMAGE=alpine:latest
+
+# derive from the appropriate base image depending on architecture
+FROM --platform=linux/amd64 ${BASEIMAGE} AS base-amd64
+FROM --platform=linux/arm64 ${BASEIMAGE} AS base-arm64
+FROM --platform=linux/arm/v7 ${BASEIMAGE} AS base-arm
+FROM --platform=linux/riscv64 riscv64/${BASEIMAGE} AS base-riscv64
+
+FROM base-${TARGETARCH} as builder
```

Comment:
```suggestion
FROM alpine:latest
```

Reply:
Thanks @Depetrol, I agree the `FROM --platform` lines can be removed. They are a vestige of when there was not a riscv64 base image, but this is now available so no need for target-specific parents.

I suggest keeping line 2 `ARG BASEIMAGE=alpine:latest` as this is common in the Docker community to allow the user to override the base image. Then line 10 would be `FROM ${BASEIMAGE} as builder`

In either case, line 25 for the application stage needs to be updated.

Reply:
Thanks for the comment! I've updated the suggestion accordingly.